docroot/themes/humsci/humsci_basic/src/scss/components/_card.scss

```diff
@@ -201,29 +201,30 @@
         // have to overwrite some Decanter things
         // to allow this to grow if no image present
         flex-grow: 1;
-        max-width: 100%;
+
+        @supports (max-width: 100%) {
+          max-width: 100%; // Apply max-width to all browsers except IE. This allows the fallback settings written in flex-column to work correctly.
```

Comment:
so weird! why does this break ie?

Reply:
I found [this article](https://alastairc.ac/2014/08/flexbox-ie11-bugs/) where they were experiencing the same bug. Since we added max-width to overwrite the styles written for `flex-column` I decided to remove it and see what happened. I was pleasantly surprised to find that things look better in IE without it.